packages/shift-react-components/src/components/search/search-bar.js

```diff
@@ -2,22 +2,37 @@
 import React, { Component } from 'react'
 import classNames from 'classnames'
 import { connectSearchBox } from 'react-instantsearch-dom'
+import debounce from 'lodash/debounce'
+
+const REFINE_DEBOUNCE_MS = 700
```

Comment:
Not to be a super pain, but I wonder if it would to our advantage of having this be used as part of `Config` and have it configurable via ENV.

I think it needs adding to next.config.js in `publicRuntimeConfig` and then configuring within `_app.js`